cloud-provider-snapshot-restore-jobs/Makefile

```diff
@@ -2,7 +2,7 @@
 
 build:
 	cfn generate
-	env GOOS=linux go build -ldflags="-s -w" -tags="logging callback scheduler" -o bin/handler cmd/main.go
+	env GOOS=linux go build -ldflags="-s -w" -tags="logging" -o bin/handler cmd/main.go
```

Comment:
Does it work without the tags, if I remember correctly the last time we tested it didn't work